kustomize/falco-driver/kmod/stress-ng.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stress-ng-driver-kmod
+  namespace: falco
+  labels:
+    app: stress-ng
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: stress-ng
+  template:
+    metadata:
+      labels:
+        app: stress-ng
+    spec:
+      nodeSelector:
+        cncf-project-sub: "falco-driver-kmod"
+      containers:
+      # https://wiki.ubuntu.com/Kernel/Reference/stress-ng
+      - name: stress-ng-1
+        image: ubuntu:22.04
+        command: ["/bin/bash", "-c"]
+        args:
+        - |
+          # apt-get update && apt-get install -y stress-ng
```

Comment:
Hi @incertum, is there a reason why is it commented in?

Reply:
oh idk anymore, thanks for the callout, let me check later!

Reply:
Fixed it, idk what happened I probably got distracted while debugging and then forgot about it, oops.
Thanks again for spotting this @maxgio92!

Reply:
Thanks to you